ElastosBountyProgram/front-end/src/module/page/developer/Component.js

```diff
@@ -115,7 +115,7 @@ export default class extends StandardPage {
                                     </div>
                                     <div className="button-container">
                                         <Button block={matches}>
-                                            <a target="_blank" href="https://github.com/elastos/Elastos.Community/tree/master/ElastosBountyProgram">
+                                            <a target="_blank" href="https://github.com/elastos/Elastos.Community/blob/3640cec31f917725db0a6e7dd686a95ac975b8c6/ElastosBountyProgram/front-end/src/module/page/developer/learn/detail/TheFourPillars/ElastosRuntime/Component.js">
```

Comment:
Why link to this file?

Reply:
hi Wiktor, I followed the requirements in Pivotal (https://www.pivotaltracker.com/n/projects/2171366/stories/160813575) as below:

> DOCS link for Elastos RunTime (RT) - C++ (on this URL: https://www.cyberrepublic.org/developer) goes to Github EBP (URL: https://github.com/elastos/Elastos.Community/tree/master/ElastosBountyProgram)
It should go to Runtime page of Github:
https://github.com/elastos/Elastos.Community/blob/3640cec31f917725db0a6e7dd686a95ac975b8c6/ElastosBountyProgram/front-end/src/module/page/developer/learn/detail/TheFourPillars/ElastosRuntime/Component.js

Reply:
@WiktorStarczewski could you discuss with Mike to see what's the right link here? thanks!